content/v2.0-preview/whats-new/_index.md

```diff
@@ -0,0 +1,236 @@
+field defaults to `Namespaced` in the new v2alpha1 version of the XRD API.
+
+```yaml
+apiVersion: apiextensions.crossplane.io/v2alpha1
+kind: CompositeResourceDefinition
+metadata:
+  name: apps.example.crossplane.io
+spec:
+  scope: Namespaced
+  group: example.crossplane.io
+  names:
+    kind: App
+    plural: apps
+  versions:
+  - name: v1
+  # Removed for brevity
+```
+
+You can also set the `scope` field to `Cluster` to create a cluster scoped XR. A
+cluster scoped XR can compose any cluster scoped resource. A cluster scoped XR
+can also compose any namespaced resource in any namespace.
+
+With namespaced XRs there's no longer a need for claims. **The new namespaced
+and cluster scoped XRs in Crossplane v2 don't support claims.**
+
+{{<hint "tip">}}
+Crossplane v2 is backward compatible with v1-style XRs.
+
+When you use v1 of the XRD API `scope` defaults to a special `LegacyCluster`
+mode. `LegacyCluster` XRs support claims and don't use `spec.crossplane`.
+{{</hint>}}
+
+## Namespaced managed resources
+
+Crossplane v2 makes all managed resources (MRs) namespaced.
+
+This enables a namespaced XR to compose entirely of namespaced resources -
+whether they're a Crossplane MR like an `RDSInstance`, a Kubernetes resource
+like a `Deployment`, or a third party custom resource like a
+[Cluster API](https://cluster-api.sigs.k8s.io) `Cluster`.
+
+A namespaced MR looks like this:
+
+```yaml
+apiVersion: s3.aws.m.upbound.io/v1beta1
+kind: Bucket
+metadata:
+  namespace: default
+  generateName: my-bucket
+spec:
+  forProvider:
+    region: us-east-2
+```
+
+Namespaced MRs work great with or without composition. Crossplane v2 isn't
+opinionated about using composition and MRs together. Namespaces enable fine
+grained access control over who can create what MRs.
+
+{{<hint "note">}}
+During the Crossplane v2 preview only namespaced AWS managed resources are
+available.
+
+<!-- vale gitlab.FutureTense = NO -->
+Maintainers will update the managed resources for other systems including Azure,
+GCP, Terraform, Helm, GitHub, etc to support namespaced MRs soon.
+<!-- vale gitlab.FutureTense = YES -->
+{{</hint>}}
+
+{{<hint "tip">}}
+Crossplane v2 is backward compatible with v1-style cluster scoped MRs.
+
+<!-- vale gitlab.FutureTense = NO -->
+New provider releases will support both namespaced and cluster scoped MRs.
+Crossplane v2 considers cluster scoped MRs a legacy feature. Crossplane will
+deprecate and remove cluster scoped MRs at a future date.
+<!-- vale gitlab.FutureTense = YES -->
+{{</hint>}}
+
+## Compose any resource
+
+Crossplane v2 isn't opinionated about using composition together with managed
+resources.
+
+You can create a composite resource (XR) that composes any resource, whether
+it's a Crossplane MR like an `RDSInstance`, a Kubernetes resource like a
+`Deployment`, or a third party custom resource like a
+[CloudNativePG](https://cloudnative-pg.io) PostgreSQL `Cluster`.
```

Comment:
we kind of said this before in the preceding namespaced managed resources section - its important, so maybe worth repeating, but was this consciously done?

Reply:
Yeah I decided to consciously repeat it. There's a small difference - the previous one mentions a ClusterAPI Cluster where as this one is a CNPG Cluster.